aiogram_dialog/context/intent_filter.py

```diff
@@ -95,7 +95,7 @@ async def on_post_process(self, data: dict):
     async def on_pre_process_aiogd_update(self, event: DialogUpdateEvent, data: dict):
         chat = get_chat(event)
```

Comment:
maybe replace all `get_chat` calls to gething chat from data: `data['event_chat']`